### ISSUE TITLE
Dev build does not fix it

### ISSUE THREAD

**Issue by rayman89** [yfdyh000/tabutils#62]
Dev build does not fix the back history menu for me on windows 7 x64 firefox version 43 b8 x64

**Comment by Floyd77** [comment]
I have the same problem with Firefox 43 final, I'm using Firefox 32 bits on Win7 x64.

**Comment by Ythan** [comment]
I also have this problem, I would love to see a fix!

**Comment by Guil211** [comment]
Same here, using FF43.0.4 on W10 x64. Any news on that front? I use back/forward history all the time, so a fix would be most appreciated. Cheers

**Comment by yfdyh000** [comment]
Sorry, though the deadline is getting closer (45 turn into Beta), but I did not further research in recent.

**Comment by Guil211** [comment]
Thanks for the quick reply. Hopefully you or someone else can look into it soon, since this impacts the browser's functionality. Otherwise I love TU, I've been using it for years now!

**Comment by Ythan** [comment]
For what it's worth, I'm using Tab Mix Plus at the moment. It offers similar functionality and does not break the history drop-down.

**Comment by Guil211** [comment]
Thanks for the tip. I reverted back to TMP for now, though it does not do tab stacking. I use this feature heavily, especially since Firefox has been moving away from Panorama. So still can't wait for a TU fix! Also, I wanted to point out that I have no problem with the back/forward history in PaleMoon with same extensions installed, even when it was running on Gecko engine. Cheers,

**Comment by rayman89** [comment]
Tree style tab and tab kit 2 do tab staking.

https://addons.mozilla.org/en-US/firefox/addon/tree-style-tab/?src=search
https://addons.mozilla.org/es/firefox/addon/tabkit-2nd-edition/?src=search

You can use both with the tabs on top so don't be put off by the default config.

Tab kit has less features but it has colored tabs tree style tab is more complete and works with multiple tab hanlder and looks like this http://www.imagebam.com/image/5c5fe1461849665  (you can change the icon + - with other icons like small aroows and so)

**Comment by Guil211** [comment]
Thanks for the tip rayman. I gave tabkit2 a try, but can't figure out how to manually stack tabs. Tree tab fits the bill much better. Looks like I'll be using it in combo with TMP until TU is back. Any news on that front? Cheers.

**Comment by rayman89** [comment]
@Guil211 on tabkit youc ant do it manually but there are some predifined auto gruping functiosn to group the actual tabs you have opened and to group the next tabs you open.

**Comment by Guil211** [comment]
@rayman89 Yes, I've seen these options, but they don't really fit the way I work, that's why I went with TreeTab.
@yfdyh000 Any news on a fix for TU? TreeTab sorta works, but I still like TU much, much more. Cheers!
